docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+# Example docker-compose for running The-Minecraft-Overviewer
+# Make sure to read it and modify to meet your needs.
+
+version: "3.8"
+services:
+  # This container changes permissions for output directory and downloads 1.20.4 jar
+  # Make sure to include custom texturepath in your overviewer config, eq. texturepath = "/mnt/minecraft/.minecraft/versions/1.20.4/1.20.4.jar"
+  prepare-files:
+    image: alpine:3.18
+    restart: "no"
+    entrypoint: |
+      /bin/sh -c "
+      mkdir -p /mnt/minecraft/.minecraft/versions/1.20.4 && 
+      chown 1000:1000 -R /usr/overviewer/output &&
+      wget https://piston-data.mojang.com/v1/objects/fd19469fed4a4b4c15b2d5133985f0e3e7816a8a/client.jar -O /mnt/minecraft/.minecraft/versions/1.20.4/1.20.4.jar && 
```

Comment:
Reasonable minds can differ on this one.

I've never been a massive fan of backporting a kubernetes style init container into a docker-compose architecture; there isn't a sensible way of enforcing ordering or the mere fact that an init container should run to completion before the main container starts.

I'm not sure if I prefer an approach of just bundling this into an entrypoint script in the main container, or if there should be an expectation for the user to grab the jar manually and bind-mount it in. Perhaps both options are reasonable here?

That said, if it's done automatically, we ought to be parameterising the client version and locating the correct URL automatically, rather than relying on this magic value in the URL.